ppr-ui/src/components/unitNotes/UnitNoteContentInfo.vue

```diff
@@ -120,8 +129,23 @@ export default defineComponent({
   components: {
     BaseAddress
   },
-  setup () {
+  setup (props) {
     const { isNoticeOfCautionOrRelatedDocType } = useMhrUnitNote()
+
+    const localState = reactive({
+      separatedRemarks: computed(() : string[] | null => {
+        if (props.note.documentType === UnitNoteDocTypes.CONTINUED_NOTE_OF_CAUTION && !props.note?.expiryDateTime) {
+          const remarks = props.note.remarks
+          const generatedRemarks = 'Continued until further order of the court. '
```

Comment:
Can be abstracted if needed in the future.

Reply:
These generated remarks could either be in a resource file or a prop text.